core/citrus-api/src/main/java/com/consol/citrus/util/IsXmlPredicate.java

```diff
@@ -0,0 +1,19 @@
+package com.consol.citrus.util;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests if a string represents a XML. An empty string is considered to be a valid XML.
+ */
+public class IsXmlPredicate implements Predicate<String> {
+    
+    public static IsXmlPredicate INSTANCE = new IsXmlPredicate();
```

Comment:
add a private constructor, this will prevent anyone from unintentionally creating a new instance.

Reply:
See above.